pkg/workers/leader_election_mgr_test.go

```diff
@@ -3,10 +3,12 @@ package workers
 import (
 	"errors"
 	"strings"
+	"sync"
 	"testing"
 	"time"
 
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	. "github.com/onsi/gomega"
```

Comment:
After the tech discussion we decided not to use `.` import alias for gomega any longer. Let's change each assert methods to use `gomega.` instead.